debian/control

```diff
@@ -34,7 +47,17 @@ Priority: extra
 Depends: memento-libs (= ${binary:Version}), gdb
 Description: Debugging symbols for memento's libraries
 
+Package: memento-nginx-node
```

Comment:
I think memento-node should just depend on memento-nginx, which means we don't need this package. (Actually, even that might be the wrong way around - I suspect the memento service's HTTP interface requires nginx integration to function properly, so perhaps memento should depend on memento-nginx...)